### ISSUE TITLE
Not allow to get token if user workspace has no icon

### ISSUE THREAD

**Issue by vitaliy-sk** [BoD/klibnotion#1]
When the user workspace doesn't have an icon the response from Notion API is following:

```
{"access_token":"xxxx","token_type":"bearer","workspace_name":"Vitaliy","bot_id":"xxxx"}
```

and parsing is failing because workspace_icon in ApiOAuthGetAccessTokenResult in mandatory

**Comment by BoD** [comment]
This has been resolved and will be part of the next release.

Thank you for reporting!

**Comment by BoD** [comment]
I've just published v1.5.1 with a fix for this issue.
